.github/workflows/builds.yml

```diff
@@ -8,33 +8,41 @@ on:
 
 env:
   CTEST_OUTPUT_ON_FAILURE: 1
-  
+
 jobs:
   builds:
     name: ${{ matrix.platform.name }}-${{ matrix.build }}
     runs-on: ubuntu-latest
-    container: ghcr.io/acts-project/ubuntu2004:v11
+    container: ${{ matrix.platform.container }}
     strategy:
       matrix:
         platform:
           - name: CPU
             container: ghcr.io/acts-project/ubuntu2004:v11
           - name: CUDA
-            container: ghcr.io/acts-project/ubuntu1804_cuda:v11
+            container: ghcr.io/acts-project/ubuntu2004_cuda:v13
         build:
           - Release
           - Debug
     steps:
       - name: Dependencies
         run: apt-get install -y git-lfs
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v1      
```

Comment:
Is there a reason to downgrade this step to version 1? Does v2 not do what we need it to do?